stacks-signer/src/chainstate.rs

```diff
@@ -56,6 +84,26 @@ pub struct SortitionState {
     pub consensus_hash: ConsensusHash,
     /// what is this signer's view of the this sortition's miner? did they misbehave?
     pub miner_status: SortitionMinerStatus,
+    /// the timestamp in the burn block that performed this sortition
+    pub burn_header_timestamp: u64,
+    /// the burn header hash of the burn block that performed this sortition
+    pub burn_block_hash: BurnchainHeaderHash,
+}
+
+/// Captures the configuration settings used by the signer when evaluating block proposals.
+#[derive(Debug, Clone)]
+pub struct ProposalEvalConfig {
+    /// How much time between the first block proposal in a tenure and the next bitcoin block
+    ///  must pass before a subsequent miner isn't allowed to reorg the tenure
```

Comment:
I think this would be a little more clear if you move "must pass" earlier, i.e. "How much time must pass between..."

Reply:
Yep, updated in [62aac67](https://github.com/stacks-network/stacks-core/pull/4956/commits/62aac67851b4104d2f12a549afaa00bbc82436c6)